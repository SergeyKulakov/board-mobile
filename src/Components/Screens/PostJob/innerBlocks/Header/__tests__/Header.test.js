import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { Header as Component } from '../index'

describe('PostJob.Header', () => {
  const defaultProps = {
    title: 'title',
    onHamburgerClick: () => null,
    onBackClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
