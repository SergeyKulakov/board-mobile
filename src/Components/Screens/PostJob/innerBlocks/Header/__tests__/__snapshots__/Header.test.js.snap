// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostJob.Header render snapshot 1`] = `
<Component>
  <Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <BackIcon
          onClick={[Function]}
        />
      </Styled(View)>
      <Component
        onClick={[Function]}
      />
    </Styled(View)>
    <Styled(View)
      flex={5}
    >
      <ScreenTitle
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        title
      </ScreenTitle>
    </Styled(View)>
    <Styled(View)
      flex={2}
    />
  </Styled(View)>
</Component>
`;
